analytics/java/src/main/java/com/koverse/examples/analytics/SentimentAnalysis.java

```diff
@@ -4,19 +4,21 @@
 import com.koverse.sdk.data.Parameter;
 import com.koverse.sdk.transform.java.DataFrameTransform;
 import com.koverse.sdk.transform.java.DataFrameTransformContext;
+
 import org.apache.spark.broadcast.Broadcast;
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.api.java.UDF1;
 import org.apache.spark.sql.functions;
 import org.apache.spark.sql.types.DataTypes;
 
+import java.io.Serializable;
 import java.util.Map;
 
 import static com.koverse.com.google.common.collect.Lists.newArrayList;
 
 
-public class SentimentAnalysis implements DataFrameTransform {
+public class SentimentAnalysis implements DataFrameTransform, Serializable {
```

Comment:
Do you need a UID when making this class Serializable? ex. @SerialVersionUID(100L)

Reply:
The short answer is yes, because even though I'm not saving a SentimentAnalysis object anywhere that would need to be deserialized, it doesn't hurt.